force_bdss/core/data_value.py

```diff
@@ -0,0 +1,34 @@
+from traits.api import HasStrictTraits, Any, String, Enum
+
+
+class DataValue(HasStrictTraits):
```

Comment:
I thought about it, we used to have originator, but I start to believe it's not useful. we can reintroduce it at a later stage with no fuss, because the setting will be done by the core anyway, so plugin authors don't have to do that manually, and nothing will change for them.